docs/modules/presence_router_callbacks.md

```diff
@@ -1,8 +1,14 @@
 # Presence router callbacks
 
-Presence router callbacks allow module developers to specify additional users (local or remote)
-to receive certain presence updates from local users. Presence router callbacks can be 
-registered using the module API's `register_presence_router_callbacks` method.
+Presence router callbacks allow module developers to define additional users
+which receive updates presence updates from local users. The additional users
+can be local or remote.
+
+For example, it could be used to have `alice` receive all presence updates from
+`bob` even though they don't share a room.
```

Comment:
```suggestion
For example, it could be used to direct all of `@alice:example.com` (a local user)'s presence updates to
`@bob:matrix.org` (a remote user), even though they don't share a room.
```

Just to demonstrate that this works over federation.

However, I believe you would need a `PresenceRouter` module on both homeservers, as while in this example `matrix.org` would receive and store `@alice:example.com`'s presence updates, Bob would not actually get Alice's presence updates in his `/sync` stream unless they shared a room, or a PresenceRouter module said he should.